### DISCUSSION TITLE
Jinja/Jinja2 accept file

### DISCUSSION THREAD

**Discussion by Rafael-Ramblas** [react-dropzone/react-dropzone#1226] [Q&A]
Hello, 
I'm building an uploader that allows the upload of jinja template files, but the browser returns me an empty string as the file type. 
The goal that I'm pursing is to maintain the effect of `isDragReject` on a list of accepted types, but allowing this type of file. 
Does anyone knows how to achieve this?

Jinja files are just python templates with the extension `.py.j2` or `.py.jinja2`
A valid jinja2 template with the name of `example-template.py.j2`:
```
#!/bin/python3
print("Hello {{planet}}")
```

File meta after upload:
```
path: "example-template.py.j2"
lastModified: 1660600158113
lastModifiedDate: Mon Aug 15 2022 18:49:18 GMT-0300 (Brasilia Standard Time) {}
name: "example-template.py.j2"
size: 42
type: ""
webkitRelativePath: ""
```

**Comment by rolandjitsu** [comment]
@Rafael-Ramblas have you seen https://react-dropzone.js.org/#section-accepting-specific-file-types ? If yes, what have you tried so far and what was the behaviour?
